features/git-ship/current_branch/on_feature_branch_without_open_changes.feature

```diff
@@ -38,8 +38,10 @@ Feature: git ship: shipping the current feature branch
   Scenario: feature branch with non-pulled updates in the repo
     Given I have a feature branch named "feature"
     And the following commit exists in my repository
-      | BRANCH  | LOCATION | FILE NAME    | FILE CONTENT    |
-      | feature | remote   | feature_file | feature content |
+      | BRANCH  | LOCATION         | FILE NAME    | FILE CONTENT          |
```

Comment:
Ha, that's why I implemented remote commits as from the normal user initially (okay, with some `git reset` madness thrown in). Then we changed it to a more clever hack that uses the co-worker repo. And now we need more hacks to work around the first hack. I think we should clean this up. Having specs behave unintuitively if I use remote commits, and forcing me to create 3 commits in that case, is not good style.